dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/SaveContentActionlet.java

```diff
@@ -64,8 +64,9 @@ public void executeAction(final WorkflowProcessor processor,
 			checkoutContentlet.setInode(inode);
 			checkoutContentlet.setProperty(Contentlet.WORKFLOW_IN_PROGRESS, Boolean.TRUE);
 
-			final Contentlet contentletNew = this.contentletAPI.checkin
-					(checkoutContentlet, processor.getContentletDependencies());
+			final Contentlet contentletNew = (null != processor.getContentletDependencies())?
+					this.contentletAPI.checkin(checkoutContentlet, processor.getContentletDependencies()):
+					this.contentletAPI.checkin(checkoutContentlet, processor.getUser(), false);
```

Comment:
@jdotcms question, if we don't pass the `getContentletDependencies` and if in the content we are trying to save we modified categories and relationships without passing the `getContentletDependencies`are we really saving those changes?

Reply:
No, we are not!